packages/manager/apps/hub/package.json

```diff
@@ -80,7 +80,7 @@
   },
   "devDependencies": {
     "@ovh-ux/manager-webpack-config": "^3.3.0 || ^4.0.0",
-    "@ovh-ux/ui-kit": "^4.4.1",
+    "@ovh-ux/ui-kit": "^4.4.3",
```

Comment:
not the scope of this PR but i don't see why ui-kit is a dev dependency here

Reply:
You're right! It doesn't make sense to have it defined as a `devDependencies`
here because it's already defined as a `dependency`.

I can't find any usage of `@ovh-ux/ovh-ui-kit` package from following location:
- `https://github.com/ovh/manager/tree/develop/packages/manager/apps/hub/src`

Except here: `https://github.com/ovh/manager/blob/master/packages/manager/apps/hub/src/index.less#L2`.

```sh
$ git grep "@ovh-ux/ui-kit" packages/manager/apps/hub/src/
packages/manager/apps/hub/src/app.module.js:import '@ovh-ux/ui-kit';
packages/manager/apps/hub/src/app.module.js:import '@ovh-ux/ui-kit/dist/css/oui.css';
packages/manager/apps/hub/src/dashboard/catalog/catalog.module.js:import '@ovh-ux/ui-kit';
packages/manager/apps/hub/src/dashboard/order-dashboard/index.js:import '@ovh-ux/ui-kit';
packages/manager/apps/hub/src/dashboard/welcome/index.js:import '@ovh-ux/ui-kit';
packages/manager/apps/hub/src/index.less:@import '@ovh-ux/ui-kit/dist/less/_tokens';
```

Tried to remove it from `devDependencies` and rebuilt the package successfully:
```sh
$ yarn workspace @ovh-ux/manager-hub-app run start
...
lerna success exec Executed command in 35 packages: "npm run build --if-present"
Done in 123.09s.
```

So I will remove it!
Thank you!